strip_attributes.gemspec

```diff
@@ -23,8 +23,8 @@ Gem::Specification.new do |spec|
     "source_code_uri"   => "https://github.com/rmm5t/strip_attributes",
   }
 
-  spec.add_runtime_dependency "activemodel", ">= 3.0", "< 8.0"
-  spec.add_development_dependency "active_attr", "~> 0.15"
+  spec.add_runtime_dependency "activemodel", ">= 3.0", "< 8.1"
+  spec.add_development_dependency "active_attr", ">= 0.15.0", "< 0.19"
```

Comment:
[QUESTION] Please help me understand why this active_attr restriction was necessary as compared to `~> 0.15` (considering the latest release of active_attr is 0.17).

Reply:
Because we need https://github.com/cgriego/active_attr/pull/205 which will (probably) be published as `active_attr` v0.18.0.

Reply:
That doesn't explain the difference between `"~> 0.15"` and `">= 0.15.0", "< 0.19"`

[PLEASE REVERT] back to `~> 0.15`